flex/validation/response.py

```diff
@@ -51,7 +52,7 @@ def validate_status_code_to_response_definition(response, operation_definition):
     except KeyError:
         raise ValidationError(
             MESSAGES['response']['invalid_status_code'].format(
-                status_code, operation_responses.keys(),
+                status_code, ', '.join(operation_responses.keys()),
```

Comment:
Doing this so that we don't see:
```
Request status code was not found in the known response codes.  Got `202`: Expected one of: `dict_key(200, 404)`
```